src/System.Management.Automation/utils/EncodingUtils.cs

```diff
@@ -102,4 +102,23 @@ public override object Transform(EngineIntrinsics engineIntrinsics, object input
 
     }
 
+    /// <summary>
+    /// To make it easier to complete the possible values for the -Encoding parameter, we add an 
+    /// unified ArgumentCompletionsAttribute here. This also solves the problem of code duplication.
```

Comment:
I think the comment should describe functionality not motivation.

Reply:
You are right. I changed it.